instrumentation/opentelemetry-instrumentation-falcon/src/opentelemetry/instrumentation/falcon/__init__.py

```diff
@@ -140,11 +140,22 @@ def instrumentation_dependencies(self) -> Collection[str]:
         return _instruments
 
     def _instrument(self, **kwargs):
+        self._original_falcon_api = None
+        self._original_falcon_app = None
```

Comment:
can we use only one attribute for both 2 and 3, and conditionally read/write falcon.App or falcon.API?

Reply:
Sure, if you prefer that